src/app/shared/services/auth/authentication.service.ts

```diff
@@ -10,7 +10,7 @@ export class AuthenticationService {
   public readonly accessToken = new BehaviorSubject<string | null>(null);
 
   public get currentRefreshToken(): string {
-    return localStorage.getItem('refreshToken') || '';
+    return localStorage?.getItem('refreshToken') || '';
```

Comment:
![image](https://github.com/anhptk/group-order/assets/27537475/92cc9e87-3835-4e47-a748-8e76333d49fd)
Always have this error when running ng serve

Reply:
This is known issue, will add it as a bug in backlog